### PR TITLE
Even more bugfixes

### PR DESCRIPTION
**Pull request by Kierany9**
Fixes https://github.com/HippieStationCode/HippieStation13/issues/2462 holy shit crystal dismemberment broke so much jesus this has been driving me up the wall for fucking ages

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -852,13 +852,13 @@
...
-		else
-			R.overlays = null
+		else if(locate(block) in R.overlays)
+			R.overlays -= block
 		if(!H.has_left_hand(1))
 			if(!(locate(block) in L.overlays))
 				L.overlays += block
-		else
-			L.overlays = null
+		else if(!locate(block) in L.overlays)
```
Comment (JohnGinnane) [inline code/modules/mob/living/carbon/human/species.dm]: I think you need to remove the `!`

Comment (Kierany9) [inline code/modules/mob/living/carbon/human/species.dm]: oops

File: html/changelogs/Kierany9 - Bugfixes2.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author: Kierany9
+
+delete-after: True
+
```
Comment (JohnGinnane) [inline html/changelogs/Kierany9 - Bugfixes2.yml]: Missing the `changes:`

**Comment by JohnGinnane** [comment]
Have you tested this?

**Comment by Kierany9** [comment]
Yes

**Comment by JohnGinnane** [comment]
Great, code looks good, merging

**Pull request merged**